examples/images_contours_and_fields/colorbar_annotation_demo.py

```diff
@@ -0,0 +1,54 @@
+"""
+=========================
+Colorbar Annotation Demo
+=========================
```

Comment:
It looks like this title underline/overline has one too many characters (it has to exactly match the title length)

Reply:
Oh, I didn't catch that! Thanks!

Reply:
I've updated this.